src/bun.js/webcore/response.zig

```diff
@@ -2015,14 +2028,22 @@ pub const Fetch = struct {
 
                         if (options.get(ctx, "tls")) |tls| {
                             if (!tls.isEmptyOrUndefinedOrNull() and tls.isObject()) {
+                                if (SSLConfig.inJS(globalThis, tls, exception)) |config| {
+                                    if (ssl_config) |existing_conf| {
+                                        existing_conf.deinit();
+                                        bun.default_allocator.destroy(existing_conf);
+                                        ssl_config = null;
+                                    }
+                                    ssl_config = bun.default_allocator.create(SSLConfig) catch @panic("out of memory!");
+                                    ssl_config.?.* = config;
+                                }
                                 if (tls.get(ctx, "rejectUnauthorized")) |reject| {
```

Comment:
we also have `rejectUnauthorized` inside SSLConfig, probably dont need this anymore